zbus/src/address/transport/nonce_tcp.rs

```diff
@@ -95,4 +85,12 @@ impl<'a> TransportTrait<'a> for NonceTcp<'a> {
 
         Ok(res)
     }
+
+    fn fmt_key_val<'s: 'b, 'b>(&'s self, kv: KeyValFmt<'b>) -> KeyValFmt<'b> {
```

Comment:
some docs about what this is all about, would be nice.

Reply:
`// add() the transport (key,val) pairs to KeyValFmt` ?

Reply:
works for me, if you also document `KeyValFmt`. :)

Reply:
done